### PR TITLE
Fix - bar button structure

### PR DESCRIPTION
**Pull request by AleksandrBu**
Test push to get comments as I ran out of ideas :)
Need to do:
- Add reusable component/approach for buttons as button's widgets do the same things
- Work on fonts when scale is decreasing

### CODE REVIEW

File: apps/codelab/src/app/codelabs/angular/angular-cli/angular-cli.component.html [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-  </div>
   <codelab-progress-bar></codelab-progress-bar>
   <slide-arrows></slide-arrows>
 
   <!--  INTRO -->
   <div *slide id="intro" no-padding>
     <codelab-title-slide
       i18n-title
-      title="angular-cli"
+      title="Angular-cli"
```
Comment (kirjs) [inline apps/codelab/src/app/codelabs/angular/angular-cli/angular-cli.component.html]: What's this?

File: apps/codelab/src/app/codelabs/angular/angular-routing.module.ts [hunk 2]
```diff
@@ -99,7 +99,7 @@ const routes = [
       {
         path: 'angular-cli',
         loadChildren: './angular-cli/angular-cli.module#AngularCliModule',
-        name: 'angular-cli',
+        name: 'Angular-cli',
```
Comment (kirjs) [inline apps/codelab/src/app/codelabs/angular/angular-routing.module.ts]: And this?

File: apps/codelab/src/app/codelabs/angular/material/material.component.html [hunk 3]
```diff
@@ -21,10 +21,6 @@
 
 
 <slide-deck slideShortcuts slidesRouting slides-tracking>
```
Comment (kirjs) [inline apps/codelab/src/app/codelabs/angular/material/material.component.html]: Are you just removing the buttons?

File: apps/codelab/src/app/codelabs/codelabs.module.ts [hunk 4]
```diff
@@ -1,23 +1,37 @@
...
+  ],
   imports: [
     BrowserAnimationsModule,
     FeedbackModule,
+    MenuShortcutModule,
+    MenuGithubModule,
     SlidesModule,
     CodelabsRoutingModule,
     FirebaseLoginModule,
+    OverlayModule
```
Comment (kirjs) [inline apps/codelab/src/app/codelabs/codelabs.module.ts]: This goes into shared module, no?

File: apps/codelab/src/app/codelabs/angular/angular-cli/angular-cli.component.html (hunk 1 above)
Comment (AleksandrBu) [reply apps/codelab/src/app/codelabs/angular/angular-cli/angular-cli.component.html]: Title with a small letter does not look good)
Yeah, it shouldn't be here) I will revert later

File: apps/codelab/src/app/codelabs/angular/angular-routing.module.ts (hunk 2 above)
Comment (AleksandrBu) [reply apps/codelab/src/app/codelabs/angular/angular-routing.module.ts]: And this)

File: apps/codelab/src/app/codelabs/codelabs.module.ts (hunk 4 above)
Comment (AleksandrBu) [reply apps/codelab/src/app/codelabs/codelabs.module.ts]: yeah, sure, makes sense

File: apps/codelab/src/app/codelabs/angular/material/material.component.html (hunk 3 above)
Comment (AleksandrBu) [reply apps/codelab/src/app/codelabs/angular/material/material.component.html]: Yeah, from here. Materail component is based on angular module which is loaded on FullLayout, which has all 4 buttons - such kind of logic)

**Pull request closed without merging**